lib/scsb-client.js

```diff
@@ -65,11 +66,16 @@ clientWrapper.getItemsAvailabilityForBarcodes = (barcodes) => scsbClient().getIt
 
 // bnum is a plain bnum without padding such as we use in the DiscoveryAPI
 clientWrapper.getItemsAvailabilityForBnum = (bnum) => {
-  const bibId = bnum.replace(/^b/, '')
-  const body = {
-    institutionId: 'NYPL',
-    bibliographicId: `.b${bNumberWithCheckDigit(bibId)}`
-  }
+  // Identify nypl-source and unprefixed id:
+  const { nyplSource, id } = NyplSourceMapper.instance().splitIdentifier(bnum)
+  // Determine SCSB "institutionId" (e.g. NYPL, HL, CUL, PUL):
```

Comment:
Or was the problem here?

Reply:
Both places you called out. The bug was:
 - we were not fetching partner SCSB statuses because the `getItemsAvailabilityForBarcodes` only handled NYPL bibs
 - we were not attempting to fold SCSB availability into partner bib item aggregations because we were miscounting partner items

Reply:
Ah so there was a hardcoded institution: 'NYPL' situation. Just an oversight? Or was there a reason for that initially?

Reply:
Just a glaring oversight